### PR TITLE
update things, get the RN benchmarks working consistently

### PR DESCRIPTION
**Pull request by quantizor**
```
Results for native
                                   styled         cssta        
---------------------------------  -------------  -------------
simple component                   0.115±0.024ms  0.012±0.002ms
prop changes                       0.247±0.024ms  0.050±0.006ms
prop shorthands                    0.190±0.023ms  0.012±0.001ms
prop shorthands with prop changes  0.403±0.034ms  0.053±0.006ms

Results for browser
                                   styled         cssta        
---------------------------------  -------------  -------------
simple component                   0.123±0.029ms  0.009±0.002ms
prop changes                       0.279±0.041ms  0.036±0.008ms
prop shorthands                    0.133±0.053ms  0.010±0.003ms
prop shorthands with prop changes  0.300±0.018ms  0.038±0.002ms
```

This sets us up to add more tests and potentially also bench other libraries like emotion etc.

### CODE REVIEW

**Comment by quantizor** [comment]
@mxstbr when you have time

**Comment by jacobp100** [comment]
Thanks for this! However, I think the benchmarks are now in the styled-components repo, and this repo is no longer used

**Comment by mxstbr** [comment]
Wait we have the benchmarks in the main repo?

**Comment by jacobp100** [comment]
I'm almost certain we did. @philpl

**Comment by jacobp100** [comment]
The plan was to do a monorepo and move this to there. https://github.com/styled-components/styled-components/pull/497

But since that's probably a way off, we could definitely clean this up. This pr looks pretty great! Should be no problem merging

File: webpack.config.js [hunk 1]
```diff
@@ -12,20 +11,12 @@ module.exports = {
...
       { test: /\.js$/, exclude: /node_modules/, use: ['babel-loader'] },
     ],
   },
-  plugins: [
-    new webpack.DefinePlugin({
-      'process.env': {
-        NODE_ENV: JSON.stringify('production'),
-      },
-    }),
-    new webpack.optimize.UglifyJsPlugin(),
```
Comment (jacobp100) [inline webpack.config.js:29]: I think these plugins do actually affect performance—especially the first! Should we leave them in for more accurate results?

**Comment by probablyup** [comment]
They're still there, I'm just using the -p webpack CLI method instead.

File: webpack.config.js (hunk 1 above)
Comment (quantizor) [reply webpack.config.js:29]: They're still there, I'm just using the `-p` [webpack CLI method](https://webpack.js.org/api/cli/#shortcuts) instead.

Comment (quantizor) [reply webpack.config.js:29]: https://github.com/styled-components/benchmark/pull/2/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R34

**Review by jacobp100 — APPROVED** [review]
LGTM

**Comment by quantizor** [comment]
@jacobp100 Could you merge, or should I just keep pushing stuff onto this branch?

**Comment by jacobp100** [comment]
Great work! Thanks!

**Comment by kitten** [comment]
@mxstbr @jacobp100 We should probably move the benchmarks to the main-repo at some point, regardless of whether we'll go for a monorepo (it looks unlikely at this point based on previous discussions I fear). But before moving it to the main repo and thus recommending a lot of people to run it, I've got two improvements that we'll need: #3 and #4

**Pull request merged**